src/neuroglancer/sliceview/vector_graphics/vector_graphics_line_renderlayer.ts

```diff
@@ -0,0 +1,236 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {ChunkState} from 'neuroglancer/chunk_manager/base';
+import {SliceView} from 'neuroglancer/sliceview/frontend';
+import {VectorGraphicsSourceOptions} from 'neuroglancer/sliceview/vector_graphics/base';
+import {MultiscaleVectorGraphicsChunkSource, RenderLayer as GenericVectorGraphicsRenderLayer, VectorGraphicsChunkSource} from 'neuroglancer/sliceview/vector_graphics/frontend';
+import {TrackableAlphaValue, trackableAlphaValue} from 'neuroglancer/trackable_alpha';
+import {TrackableFiniteFloat, trackableFiniteFloat} from 'neuroglancer/trackable_finite_float';
+import {mat4, vec3} from 'neuroglancer/util/geom';
+import {Buffer} from 'neuroglancer/webgl/buffer';
+import {GL_ARRAY_BUFFER, GL_FLOAT} from 'neuroglancer/webgl/constants';
+import {makeTrackableFragmentMain, makeWatchableShaderError, TrackableFragmentMain} from 'neuroglancer/webgl/dynamic_shader';
+import {ShaderBuilder, ShaderProgram} from 'neuroglancer/webgl/shader';
+
+export const FRAGMENT_MAIN_START =
+    '//NEUROGLANCER_VECTORGRAPHICS_LINE_RENDERLAYER_FRAGMENT_MAIN_START';
+
+const DEFAULT_FRAGMENT_MAIN = `void main() {  
+vec3 color = vec3(0,1,0);
+  
+  float distance = length(vNormal);
+  
+  float feather = 0.5;
+  
+  if (distance >= 1.0 - feather) {
+    emitRGBA(vec4(color, (distance - 1.0) / -feather )); 
+  }
+  else if (distance < 1.0 - feather) {
+  	emitRGB(color);
+  }
+  else {
+    emitTransparent();
```

Comment:
You're right -- it isn't actually required because of the way the other two cases are setup. Originally the antialiasing code worked a little bit differently (or, perhaps, didn't work but was different...).